app/uk/gov/hmrc/helptosavefrontend/connectors/NSIConnector.scala

```diff
@@ -19,14 +19,16 @@ package uk.gov.hmrc.helptosavefrontend.connectors
 import javax.inject.{Inject, Singleton}
 
 import com.google.inject.ImplementedBy
-import play.api.{Configuration, Logger}
+import play.api.{Application, Configuration, Logger}
 import play.api.http.Status
 import play.api.libs.json.{Format, Json}
 import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig.{nsiAuthHeaderKey, nsiBasicAuth, nsiUrl}
-import uk.gov.hmrc.helptosavefrontend.config.WSHttpProxy
+import uk.gov.hmrc.helptosavefrontend.config.{FrontendAuditConnector, WSHttpProxy}
 import uk.gov.hmrc.helptosavefrontend.connectors.NSIConnector.{SubmissionFailure, SubmissionResult, SubmissionSuccess}
-import uk.gov.hmrc.helptosavefrontend.models.NSIUserInfo
+import uk.gov.hmrc.helptosavefrontend.models.{ApplicationSubmittedEvent, NSIUserInfo}
+import uk.gov.hmrc.helptosavefrontend.util.HTSAuditor
 import uk.gov.hmrc.helptosavefrontend.util.HttpResponseOps._
+import uk.gov.hmrc.play.audit.http.connector.AuditConnector
 import uk.gov.hmrc.play.http._
```

Comment:
i think we are not using `Application` ,  `FrontendAuditConnector`   and `uk.gov.hmrc.play.audit.http.connector.AuditConnector`  anymore in this class?